src/uct/ib/rdmacm/rdmacm_cm.c

```diff
@@ -61,6 +61,51 @@ ucs_status_t uct_rdmacm_cm_reject(struct rdma_cm_id *id)
     return UCS_OK;
 }
 
+ucs_status_t uct_rdmacm_cm_get_cq(uct_rdmacm_cm_t *cm, struct ibv_context *verbs,
+                                  const char *device_name, struct ibv_cq **cq)
+{
+    khiter_t iter;
+    int ret;
+
+    iter = kh_put(uct_rdmacm_cm_cqs, &cm->cqs, device_name, &ret);
+
+    if (ret == -1) {
+        ucs_error("cm %p: cannot allocate hash entry for CQ", cm);
+        return UCS_ERR_NO_MEMORY;
+    }
+
+    if (ret == 0) {
+        *cq = kh_value(&cm->cqs, iter);
+        return UCS_OK;
+    }
+
+    /* Create a dummy completion queue */
+    *cq = ibv_create_cq(verbs, 1, NULL, NULL, 0);
+    if (cq == NULL) {
```

Comment:
i suggest to use local variable for cq (and rename func param to cq_p)
then, you would see the bug here :)